### PR TITLE
green_screen and invisitibity clock

### PR DESCRIPTION
**Pull request by HadBeen**
AAAAAAAAAAAAAAAAAAA why bs7

### CODE REVIEW

File: detect_by_bolor_vid.py
Comment (Mounir-charef) [inline detect_by_bolor_vid.py:1]: use the cv2 bgr2hsv not the one from cv1

File: green_screen.py
Comment (Mounir-charef) [inline green_screen.py:1]: remove  l commented code hdk

File: hadjer.py
Comment (Mounir-charef) [inline hadjer.py:1]: if its working put it in filters.py

File: invisibility_cloak1.py
Comment (Mounir-charef) [inline invisibility_cloak1.py:1]: if bitwise_not and add_weighted are working put them if tools and remove the commented code

**Comment by RedhaWassim** [comment]
HHHHHHHHHHHHHHHHHH 0/20

**Review by Mounir-charef — COMMENTED** [review]
Fix those changes first

**Review by Mounir-charef — APPROVED** [review]

**Pull request merged**